k8s/prod/common/ia/case-api.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: case-api
+  namespace: ia
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: ia-case-api
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ia-case-case-api
+    version: 0.0.3
+  values:
+    java:
+      replicas: 2
+      memoryRequests: '768Mi'
```

Comment:
don't need any of these memory / cpu requests and limits

Reply:
I will remove memory / cpu limits. I assume some defaults applied